models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -48,8 +55,28 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
     sub.sipnet.output <- subset(sipnet.output, year == y)
     sub.sipnet.output.dims <- dim(sub.sipnet.output)
     dayfrac <- 1 / out.day
-    step <- seq(0, 0.99, dayfrac)
-
+    step <- seq(0, 0.99, dayfrac)  ## probably dont want to use hard-coded "step" because leap years may not contain all "steps", or
```

Comment:
Same comment as in ED -- use `head(seq(0, 1, by = dayfrac), -1)`. Also, I personally like to explicitly name the `by` argument for `seq`, but that's just me.

Reply:
will update, thanks!